Formula/openjdk@8.rb

```diff
@@ -102,32 +107,44 @@ def install
       --with-vendor-name=#{tap.user}
       --with-vendor-url=#{tap.issues_url}
       --with-vendor-vm-bug-url=#{tap.issues_url}
+      --with-giflib=system
     ]
 
+    ldflags = ["-Wl,-rpath,#{loader_path.gsub("$", "\\$$")}/server"]
```

Comment:
Will need to check what is needed to get this properly escaped and passed down to linker.